### ISSUE TITLE
Toolbar Editor - Make preferences the same as the fixed Topbar Toolbar

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#4142]
![bforartists_cVIdgprWEg](https://github.com/Bforartists/Bforartists/assets/35357409/0bee6cb1-ddba-4cb7-b9f0-667e235452b0)

@Blenux did a nice job of remaking the toolbar so there is a quick hand to configure all the toolbars, then the sub-toolbars also have a notice on the modes and usecase. 

**Pros:**
- You can click and slide to activate, deactivate
- You can quick hand configure the whole bar from the same place
- You can also now turn on the quick toggles and choose which toolbar type you'd like
- You can reset to default settings

So.. all these cool new features should also go to the Toolbar Editor system. 

## 
Postponed for now. Waiting on pull of #3979

**Comment by ReinerBforartists** [comment]
This looks really nice :)

**Comment by Blenux** [comment]
Plan to do this soon, probably for the 5.0 release?

**Comment by Draise14** [comment]
Got 3 months, totally doable. This version jump is so crazy.

**Comment by Draise14** [comment]
Though 4.5 is in a month gimme to think of it, if you think it can be done sooner

**Comment by Blenux** [comment]
will try to get this in for 5.0 release (planned today but yeah been a crazy day)

**Comment by Blenux** [comment]
In progress

https://github.com/user-attachments/assets/8e1e953d-41fd-466a-b796-7a4d1e659c09

**Comment by Draise14** [comment]
Oh yeaaaah!

**Comment by Blenux** [comment]
Found a issue, need to rethink my current approach.
